src/main/java/bio/terra/service/dataset/flight/ingest/IngestUtils.java

```diff
@@ -288,8 +292,17 @@ public static long countAndValidateBulkFileLoadModelsFromPath(
                             return Stream.empty();
                           }
                         })
-                    .map(n -> objectMapper.convertValue(n, BulkLoadFileModel.class)))
-        .distinct();
+                    .map(
+                        n -> {
+                          try {
+                            return objectMapper.convertValue(n, BulkLoadFileModel.class);
+                          } catch (IllegalArgumentException ex) {
+                            errorCollector.record("Error: %s", ex.getMessage());
+                          }
+                          return null;
+                        }))
+        .distinct()
+        .filter(Objects::nonNull);
```

Comment:
Since we don't use the same error mapper for combined ingests, it will throw an error if it doesn't recognize one of the fields provided. We should catch that and add it to the error details we return.